eth2near/eth2near-block-relay/index.js

```diff
@@ -184,35 +196,42 @@ class Eth2NearRelay {
           for (let i = clientBlockNumber + 1; i <= endBlock; i++) {
             if (this.validateHeaderMode === 'bsc') {
               const block = await this.robustWeb3.getBlock(i)
+              if (await isBlockAlreadySubmitted(block)) {
+                if (endBlock <= chainBlockNumber) {
+                  endBlock++
+                  continue
+                }
+              }
```

Comment:
Why do we need such error-prone (IMO) construction? I believe we can just have it like this:

```suggestion
              if (await isBlockAlreadySubmitted(block)) {
                continue
              }
```

Reply:
Changed f3c20251c33750280ab677256c6c7f6da1f438fc